doc/README.open_ce_test.md

```diff
@@ -116,16 +116,16 @@ optional arguments:
                         current working directory. (default: )
   --python_versions PYTHON_VERSIONS
                         Comma delimited list of python versions to build for ,
-                        such as "3.8" or "3.9" or "3.10". (default: 3.10)
+                        such as "3.8" or "3.9" or "3.10". (default: 3.10 when --build_types=cuda 11.8 or cpu) (default: 3.9 when --build_types=cuda 11.2) 
```

Comment:
This change is just for the documentation. Even in functionality we want to change DEFAULT_PY_VERSION to 3.9 or 3.10 based on build_type and cuda version . https://github.com/ArchanaShinde1/open-ce-builder/blob/update-badges/open_ce/constants.py#L22